<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TriangleViewer</class>
 <widget class="QMainWindow" name="TriangleViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TriangleViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="1,0,1">
    <item row="0" column="2">
     <widget class="QFrame" name="infoFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="sidesLabel">
         <property name="text">
          <string>Sides</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sideALabel">
         <property name="text">
          <string>Side a:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sideBLabel">
         <property name="text">
          <string>Side b:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sideCLabel">
         <property name="text">
          <string>Side c:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QFrame" name="settingFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="setSidesLabel">
         <property name="text">
          <string>Set sides</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Side a</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sideAField"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Side b</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sideBField"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Side c</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sideCField"/>
       </item>
       <item>
        <widget class="QPushButton" name="setSidesButton">
         <property name="text">
          <string>Set sides</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QFrame" name="increaseFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0">
       <item>
        <widget class="QLabel" name="increaseLabel">
         <property name="text">
          <string>Increase sides</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="increaseField"/>
         </item>
         <item>
          <widget class="QPushButton" name="increaseButton">
           <property name="text">
            <string>Increase</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="calculationsFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="infoLabel">
         <property name="text">
          <string>Information</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="perimeterLabel">
         <property name="text">
          <string>Perimeter:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="areaLabel">
         <property name="text">
          <string>Area:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="anglesLabel">
         <property name="text">
          <string>Angles:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mediansLabel">
         <property name="text">
          <string>Medians:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="rectangularLabel">
         <property name="text">
          <string>Is rectangular:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QFrame" name="imageFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1,0">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Image</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ViewWidget" name="viewWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QCheckBox" name="sidesCheckBox">
           <property name="text">
            <string>Show sides</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="mediansCheckBox">
           <property name="text">
            <string>Show medians</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="verticesCheckBox">
           <property name="text">
            <string>Show vertices</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>554</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>New</string>
    </property>
    <addaction name="actionTriangle"/>
   </widget>
   <addaction name="menuNew"/>
  </widget>
  <action name="actionTriangle">
   <property name="text">
    <string>Triangle</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewWidget</class>
   <extends>QWidget</extends>
   <header location="global">viewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
